experimental/GITFans/GITFans.jl

```diff
@@ -0,0 +1,601 @@
+"""
+  GITFans is a collection of Julia functions for computations
+  like those shown in the paper
+  [J. Boehm, S. Keicher, Y. Ren:
+  Computing GIT-Fans with Symmetry and the Mori Chamber Decomposition of M06bar](https://arxiv.org/abs/1603.09241).
+  ([Math. Comp. 89 (2020),
+  3003-3021](https://mathscinet.ams.org/mathscinet-getitem?mr=4136555))
+"""
+module GITFans
+
+# the necessary Julia packages
+using Oscar
+
+# for using LaTex in docstrings processed by Documenter.jl
+import Markdown
+
+export git_fan
+
+#############################################################################
+
+# hack until characteristic zero matrix groups are available
+
+import Oscar: general_linear_group
+
+function Oscar.general_linear_group(T::Type{Oscar.MatrixGroup}, n::Int, ::Oscar.FlintIntegerRing)
+    G = MatrixGroup(n, ZZ)
+    G.X = GAP.Globals.GL(Oscar._gap_filter(T), n, GAP.Globals.Integers)
+    return G
+end
+
+#############################################################################
+
+# utility functions
+
+## a <= b for two arrays `a`, `b` of bitsets,
+## defined lexicographically
+function less_or_equal_array_bitlist(a::Vector{BitSet}, b::Vector{BitSet})
+    for i in 1:length(a)
+        if a[i].bits < b[i].bits
+            return true
+        elseif a[i].bits > b[i].bits
+            return false
+        end
+    end
+    return true
+end;
```

Comment:
```suggestion
end
```
Several similar instances of redundant semicolons should also be updated